### PR TITLE
docs: update README.md to add WeChat public number, video number of QR code pictures.

### PR DESCRIPTION
**Pull request by wang-yeliang**
### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Comment by juzhiyuan** [comment]
Hi @wang-yeliang, please fill this PR's description.

- What this PR does?
- Check the Checklist

File: docs/zh/latest/README.md [hunk 1]
```diff
@@ -39,30 +39,45 @@ Apache APISIX 的技术架构如下图所示：
 ## 社区
 
 - 邮件列表 - 发送任意内容到 dev-subscribe@apisix.apache.org 后，根据回复以订阅邮件列表。
+
```
Comment (Yiyiyimu) [inline docs/zh/latest/README.md:42]: Might need to tweak IDE formatting tools or just turn it off temporarily, so unrelated changes like this won't be committed

**Review by spacewander — CHANGES_REQUESTED** [review]
Please remove extra whitespace and blank lines:
https://github.com/apache/apisix/pull/4184/checks?check_run_id=2510197574
https://github.com/apache/apisix/pull/4184/checks?check_run_id=2510197671

**Review by juzhiyuan — APPROVED** [review]

**Review by spacewander — APPROVED** [review]
